pokemongo_bot/__init__.py

```diff
@@ -295,7 +295,10 @@ def print_player_data(self, player):
         stardust = player['currencies'][1].get('amount', '0')
 
         logger.log('[#]')
-        logger.log('[#] Username: {username}'.format(**player))
+        if self.config.ci is False:
```

Comment:
I would disable this area's output - rather than just hiding the username.

Creation date is super unique + location, which would make identifying the test bot easy.